### ISSUE TITLE
Navigation section page and second level nav structure

### ISSUE THREAD

**Issue by vic-topcoder** [topcoder-platform/community-app#2403]
**Describe the bug**
When a section is selected in the first-level navigation (a section has its own second-level navigation), the main page of the section should be listed and selected in the second-level navigation.

**To Reproduce**

- Select a section;
- A page is loaded, and and the first second level item appears selected (bold, underlined).
- Click on the selected item
- A new page is loaded

**Expected behavior**
- A selected item in the second-level navigation should not be clickable
- All pages that are part of first-level section should be listed.

Check https://marvelapp.com/f9b88ee/screen/50064026 and click on few of the second-level items. 
Check how the navigation works when you click on COMMUNITY and Competitive Programming.
Observe that a selected item is not clickable.

**Comment by Oanh-and-only-Oanh** [comment]
@nithyaasworld is this the same as https://github.com/topcoder-platform/community-app/issues/2408

**Comment by nithyaasworld** [comment]
No @Oanh-and-only-Oanh They are different.

**Comment by khushimehta03** [comment]
@luizrrodrigues  kindly assign this to me till then I will enable 2 factor authentication and get added in group again.

**Comment by luizrrodrigues** [comment]
Assigned to @khushimehta03

**Comment by luizrrodrigues** [comment]
@khushimehta03 Please make sure remove hover status from mouse pointer when a main menu or secondary menu are selected. ref #2431

**Comment by luizrrodrigues** [comment]
@khushimehta03 Any status on this?

**Comment by nahidshahin** [comment]
@luizrrodrigues can I pick it?

**Comment by luizrrodrigues** [comment]
@nahidshahin Yes, before assign to this one, make sure reply feedback of other issues.

Thanks.

**Comment by gets0ul** [comment]
@luizrrodrigues Is this open for pickup? Can I work on this?

**Comment by luizrrodrigues** [comment]
@gets0ul Yes, assigned to @gets0ul

**Comment by gets0ul** [comment]
PR: https://github.com/topcoder-platform/navigation-component/pull/21

**Comment by gets0ul** [comment]
PR: https://github.com/topcoder-platform/navigation-component/pull/22

**Comment by luizrrodrigues** [comment]
@gets0ul `onClick` solution not will be good, because we working to fix nav to always showing when user is logged, so wen we refresh the page or access page direct to link, `onClick` not will be called.

Make sense?

**Comment by gets0ul** [comment]
@luizrrodrigues  
PR is updated without using onClick.  
https://github.com/topcoder-platform/navigation-component/pull/22

**Comment by luizrrodrigues** [comment]
@gets0ul Cool.
I see an issue, **WORK** will have own secondary menu, so when click in **WORK** will show bellow the secondary menu with Dashboard, My Profile e Payments (we running this in another issue)

- Work
    - Dashboard
    - My Profile
    - Payments
- Compete
    - All challenges
    - Competi...
- Tracks
    - Competitive
    - Data Science
    ...

The issue is:
- Click in Work;
- Click in Compete;
- Click in Work (this need works)

**Comment by luizrrodrigues** [comment]
@gets0ul Please hold on this, getting confirmation.

**Comment by luizrrodrigues** [comment]
@gets0ul Confirmed, we need able to back to Work menu: https://github.com/topcoder-platform/community-app/issues/2403#issuecomment-503526549

**Comment by gets0ul** [comment]
@luizrrodrigues PR updated.
